src/specifications/profile-resolution/profile-resolution-working-specml.xml

```diff
@@ -0,0 +1,1794 @@
+    </section>
+  </section>
+  <section id="target-structure">
+    <head>Target catalog structure</head>
+    <p>Inasmuch as the target of profile resolution is a catalog, the resulting document is expected
+      to be a catalog document as specified by OSCAL, conforming to the applicable schema, XSD in
+      the case of OSCAL XML or JSON Schema for OSCAL object serialization formats.</p>
+    <p>These two approaches to validation do not constrain their respective data models – although
+      OSCAL-equivalent – in exactly the same way. In particular, element structures in the XML may
+      be represented in JSON by either objects, or arrays of objects, depending on the respective object's
+      OSCAL definition; and where objects appear outside arrays, ordering semantics cannot be respected (whereas in XML, order among nodes is
+      always an intrinsic property of the graph).</p>
+    <p>Consequently, where these specifications dictate that targets appear in order, it must be
+      understood that what is described is <em>both</em> (a) the order in XML, and (b) the <term>canonical
+        order</term> in OSCAL, which while it is not exposed in a JSON object serialization, will
+      always available to an OSCAL processor by reference to the appropriate metaschema (in this
+      case, the Metaschema instance that defines the catalog model).</p>
+    <section id="form-of-result">
+      <head>Form and organization of resolution target</head>
+      <p>The output of a profile resolution should take the form of a catalog.</p>
+      <p>In most cases, catalog results (the output of a conformant profile resolution)
+        will be valid to the OSCAL catalog schema. <revisit>Where
+          invalid results are possible, this specification should note the possibility and the
+          expected variance. There are cases where the <em>target</em> of transformation is actually
+          invalid (including trivial cases of profiles whose links all fail to resolve). <xref rid="validation-of-result"/></revisit></p>
+      <p> At its base, an OSCAL <src>profile</src> source produces a <tgt>catalog</tgt>:</p>
+      <mapping unit-test="base-test_profile.xml">
+        <tagging whose="source_profile">&lt;profile id="profile-identifier">...&lt;/profile></tagging>
+        <tagging whose="target_catalog">&lt;catalog id="<hi>{ new-catalog-identifier }</hi>">
+  &lt;metadata>[Required metadata, as described below]&lt;/metadata>
+  <hi>{ controls or control groups, as described below }</hi>
+  &lt;back-matter><hi>{ back matter as described below }</hi>&lt;/back-matter>
+&lt;/catalog></tagging>
+      </mapping>
+      <p>A valid catalog must have <tgt>metadata</tgt> in addition to controls and control groups.
+        Additionally it may have <tgt>back-matter</tgt>. How to produce and populate the
+          <tgt>metadata</tgt> and <tgt>back-matter</tgt> is described below 
+        <revisit>link me</revisit>, as is the construction
```

Comment:
TODO.

Reply:
More work on the spec is in order: this is pretty soft.